### DISCUSSION TITLE
[Request] Fancy frames for QR Codes

### DISCUSSION THREAD

**Discussion by dagronf** [dagronf/QRCode#51] [Ideas]
Request for [fancy frame support](https://github.com/dagronf/QRCode/issues/50#issue-2506869904) :-

As suggested by @Bob-GGB (https://github.com/Bob-GGB)

the frame style link: https://www.qr-code-generator.com/
As shown in this picture：

![image](https://github.com/user-attachments/assets/f1e8e0ec-c19e-4020-9499-2597dd1c9b8e)

**Comment by dagronf** [comment]
See also [Add support for frames](https://github.com/dagronf/QRCode/issues/65) by @vicvoh

**Comment by shiny** [comment]
Exactly what I need. I'm looking for an alternative way to achieve this effect.
